src/librustc_codegen_utils/codegen_backend.rs

```diff
@@ -21,6 +21,7 @@ pub use rustc_data_structures::sync::MetadataRef;
 
 pub trait CodegenBackend {
     fn init(&self, _sess: &Session) {}
+    fn after_parsing(&self, _sess: &Session) {}
```

Comment:
This function is never called.

Reply:
Yeah, I forgot to actually add the call to it. I'm pushing a fix once I verify that emscripten now works.